myocamlbuild.ml

```diff
@@ -6,10 +6,10 @@ open Ocamlbuild_plugin;;
 
 let env = BaseEnvLight.load() (* setup.data *)
 
-let fortran =
-  try BaseEnvLight.var_get "fortran" env with _ -> failwith "XXX"
+let fortran = try BaseEnvLight.var_get "fortran" env with _ -> failwith "XXX"
```

Comment:
On Tue, 9 Jun 2015 07:51:02 -0700, Leonid Rozenberg wrote:
> 
> In myocamlbuild.ml:
> 
> > @@ -6,10 +6,10 @@ open Ocamlbuild_plugin;;
> >
> >  let env = BaseEnvLight.load() (* setup.data *)
> >
> > -let fortran =
> > -  try BaseEnvLight.var_get "fortran" env with _ -> failwith "XXX"
> > +let fortran = try BaseEnvLight.var_get "fortran" env with _ -> failwith "XXX"
> 
> A holdout from when I was looking through the file, I found the one liner more
> readable. If you don't like the change, happy to remove it.

I do not have strong feelings; if you feel it is more readable, go ahead.